operations/patch_list.go

```diff
@@ -64,13 +72,35 @@ func PatchList() cli.Command {
 				}
 			}
 
+			if outputJSON {
+				display := make([]restModel.APIPatch, len(patches)-1)
```

Comment:
I think you want to set the size of the slice to 0, and the capacity (third argument to make) to len(patches). 

Though, frankly, I think for cases like this, I think making a slice without a size, and just using append is fine (and make in cases like this feels like a premature optimization.) 

Out of curiosity, though I don't get why you'd want len-1?

Reply:
Because I was misusing make and it gave me a bad result.

Reply:
I have made this change.